docs/writing-rules/experiments/aliengrep.md

```diff
@@ -127,9 +129,9 @@ YAML syntax makes it easy to introduce significant newline characters in pattern
 
 ### Long ellipsis (`....`)
 
-A long ellipsis `....` and its capturing variant `$....X` matches a sequence of any lexical elements even in single-line mode. It's useful for skipping any number of lines in single-line mode.
+A long ellipsis is written as four dots `....` and its capturing variant `$....X` matches a sequence of any lexical elements even in single-line mode. It's useful for skipping any number of lines in single-line mode.
```

Comment:
```suggestion
A long ellipsis (written as four dots, `....`) and its capturing variant `$....X` matches a sequence of any lexical elements even in single-line mode. It's useful for skipping any number of lines in single-line mode.
```
The "is" verb forms a complete clause at the start of the sentence, so I removed it so that the subject of the sentence (long ellipsis and capturing variant) remains clear.